src/cmd/build.js

```diff
@@ -56,8 +92,16 @@ async function defaultPackageCreator(
     filter: (...args) => fileFilter.wantFile(...args),
   });
 
+  let extensionName: string = manifestData.name;
+
+  if (manifestData.default_locale) {
+    let messageFile = path.join(sourceDir, '_locales',
+                            manifestData.default_locale, 'messages.json');
```

Comment:
only indent new blocks by two spaces unless you're aligning it with an opening paren or brace.